src/pages/index.js

```diff
@@ -43,6 +46,40 @@ const simple_step_content = [
     },
 ]
 const Home = () => {
+    /* redirect livechat for en to open live chat popup */
+    let script_timeout = null
+    const live_chat_key = 'live_chat_redirection'
+    const [is_mounted, setMounted] = useState(false)
+    const lang = getLanguage()
+    const live_chat_enable = getLiveChatStorage()
+
+    const checkLiveChatRedirection = () => {
+        if (lang == 'en') {
+            if (live_chat_enable) {
+                removeLocalStorage(live_chat_key)
+                navigate(live_chat_redirection_link, { replace: true })
+            } else {
+                removeLocalStorage(live_chat_key)
+            }
+        } else {
+            removeLocalStorage(live_chat_key)
+        }
+    }
```

Comment:
```suggestion
        removeLocalStorage(live_chat_key)
        if (lang == 'en' && live_chat_enable) {
            navigate(live_chat_redirection_link, { replace: true })
       }
    }
```
since `removeLocalStorage(live_chat_key)` is called in all condition, i think this would be a cleaner statement